modules/apps/click-to-chat/click-to-chat-web/src/main/resources/META-INF/resources/dynamic_include/view.jsp

```diff
@@ -16,4 +16,70 @@
 
 <%@ include file="/dynamic_include/init.jsp" %>
 
-<liferay-util:include page='<%= "/dynamic_include/" + clickToChatChatProviderId + ".jsp" %>' servletContext="<%= application %>" />
\ No newline at end of file
+<liferay-util:include page='<%= "/dynamic_include/" + clickToChatChatProviderId + ".jsp" %>' servletContext="<%= application %>" />
+
+<style type="text/css">
+	.click-to-chat-hidden {
+		display: none !important;
+	}
+</style>
+
+<script type="text/javascript">
+    (function () {
+        function changeVisible(selectors, show) {
+            let selectorsList = selectors.split(',')
+            if(show) {
```

Comment:
SF is missing for this entire file, in some cases we have if(condition) everything together, spaces are missing

if (show) { }

instead of if(show)